src/components/Cell/Cell.tsx

```diff
@@ -1,23 +1,49 @@
 import { useState } from 'react';
+import { useDrop } from 'react-dnd';
 import './Cell.scss'
 
 type GridCell = {
     id: string;
 }
 
+type CardProps = {
+    plant: {
+        id: number
+        name: string
+        image: string
+        type: string
+        sunlight: string[]
+        hardiness: string
+    }
+}
+
 const Cell = ({id}: GridCell) => {
     const [isDisabled, setIsDisabled] = useState<boolean>(false)
+    const [cellContents, setCellContents] = useState<CardProps>()
+    const [{ isOver }, dropRef] = useDrop({
+        accept: 'plant',
+        drop: (plant: CardProps) => isDisabled ? console.log('fail') : setCellContents(plant),
```

Comment:
I would suggest to error handle differently than a console.log. but otherwise, great job!